source/_components/hive.markdown

```diff
@@ -57,6 +62,62 @@ scan_interval:
   default: 2
 {% endconfiguration %}
 
+## Service - Boost Heating
+
+You can use the service `hive.boost_heating` to set your heating to boost for a period of time at a certain target temperature".
+
+| Service data attribute | Optional | Description                                                            |
+| ---------------------- | -------- | ---------------------------------------------------------------------- |
+| `entity_id`            | no       | String, Name of entity e.g. `climate.heating`.                         |
+| `time_period`          | no       | Time Period, Period of time the boost should last for e.g. `01:30:00`. |
+| `temperature`          | yes      | String, The required target temperature e.g. `20.5`                    |
+
+Examples:
+
+```yaml
+# Example script to boost heating, boost period and target temperature specified.
+script:
+  boost_heating:
+    sequence:
+      - service: hive.boost_heating
+        data:
+          entity_id: "climate.heating"
+          time_period: '01:30:00'
+          temperature: "20.5"
+```
+
+## Service - Boost Hotwater
+
+You can use the service `hive.boost.hotwater` to set your hot water to boost for a period of time.
+
+| Service data attribute | Optional | Description                                                            |
+| ---------------------- | -------- | ---------------------------------------------------------------------- |
+| `entity_id`            | no       | String, Name of entity e.g. `water_heater.hot_water`.                  |
+| `time_period`          | yes      | Time Period, Period of time the boost should last for e.g. `01:30:00`. |
+| `on_off`               | no       | String, The mode to set the boost to on or odd e.g `on`                |
+
+Examples:
+
+```yaml
+# Example script to boost hotwater, boost period specified
+script:
+  boost_hotwater:
+    sequence:
+      - service: "hive.boost_hotwater"
```

Comment:
`hotwater` is actually not a valid dictionary word, `hot water` is. I would have expected the service to be `hive.boost_hot_water`.